app.rb

```diff
@@ -15,6 +19,58 @@ def list_all_books
     @option.list_options
   end
 
+  def save_book(title, author)
+    File.open('books.txt', 'a') do |f|
+      f.write "{ \"title\": \"#{title}\", \"author\": \"#{author}\"} \n"
+    end
+  end
+
```

Comment:
- [ ] Books data are persisting in a `.txt` file instead of a _json_ file. Let's ensure all data are saved in a `.json` file.

**Books data persisting in `.txt` instead of `.json`:**
![image](https://user-images.githubusercontent.com/12745474/234943236-36c4b0ff-47a3-49cb-87e4-eea483fc6011.png)